editor/debugger/script_editor_debugger.cpp

```diff
@@ -807,6 +807,10 @@ void ScriptEditorDebugger::_parse_message(const String &p_msg, uint64_t p_thread
 	} else if (p_msg == "request_quit") {
 		emit_signal(SNAME("stop_requested"));
 		_stop_and_notify();
+	} else if (p_msg == "request_embed_suspend_toggle") {
+		emit_signal(SNAME("embed_suspend_toggle_requested"));
+	} else if (p_msg == "request_embed_next_frame") {
+		emit_signal(SNAME("embed_next_frame_requested"));
```

Comment:
Adding a signal for each action is not very scalable. You could have one signal for embed actions and pass an enum value.

Reply:
Makes sense. I made the signal use an enum now. 
I'm just not too sure if I used it right.